other/nightly/install.sh

```
@@ -27,6 +36,144 @@ ROOT_USERNAME=${ROOT_USERNAME:-}
+    DOCKER_POOL_BASE_PROVIDED=true
+fi
+
+if [ -n "${DOCKER_ADDRESS_POOL_SIZE+x}" ]; then
+    DOCKER_POOL_SIZE_PROVIDED=true
+fi
+
+restart_docker_service() {
+    # Check if systemctl is available
+    if command -v systemctl >/dev/null 2>&1; then
+        systemctl restart docker
+        if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)